tests/_support/AcceptanceTester.php

```diff
@@ -68,6 +71,43 @@ public function generateEvent() {
 		return array_merge( self::$defaultEvent, $random );
 	}
 
+	/**
+	 * Update Event
+	 */
+	public function edit_event( $updateEvent )  {
+		$originalTitle		= $updateEvent['originalTitle'];
+		$newTitle			= $updateEvent['newTitle'];
+		$content			= $updateEvent['content'];
+		$allDay 			= $updateEvent['allDay'];
+		if(empty($updateEventp['originalTitle']) || empty($updateEvent['newTitle'])){
+            throw new \InvalidArgumentException('Title missing');
+        }
+
+		$I = $this;
+
+		$I->amOnPage( '/wp-admin/edit.php?post_type=tribe_events' );
+		$I->fillField( 's', $originalTitle );
+		$I->click( '#search-submit' );
+		$I->click( $originalTitle );
+
+		//We should now be on the edit page but check
+		//$I->see( $updateEvent['originalTitle'] );
```

Comment:
This and the other `$I->see()` on line 102 are commented out - is there a reason for that/do they currently fail and we're going to address later?